roles/container-engine/cri-o/defaults/main.yml

```diff
@@ -5,4 +5,4 @@ crio_seccomp_profile: "/etc/crio/seccomp.json"
 
 crio_cgroup_manager: "{{ kubelet_cgroup_driver | default('cgroupfs') }}"
 
-crio_runc_path: "/usr/sbin/runc"
+crio_runc_path: "/usr/bin/runc"
```

Comment:
This is probably the wrong place to set the variable. But what would be the right place?

Reply:
OS specific variables are in `vars/*.yml`, I don't think it's necessary to change the default value for all OS platforms.

Reply:
This is not in the change set any more (use 'Files changed' tab).